cmd/prom2hny/main.go

```diff
@@ -88,6 +85,43 @@ func NewMetricGroups(mfs []*dto.MetricFamily) []*MetricGroup {
 	return metricGroups
 }
 
+// We can't use m.GetGauge().GetValue() because there exist some
+// exceptional cases we need to handle. Abstract exception cases
+// in this func
+func getDatapointFromMetric(mf *dto.MetricFamily, m *dto.Metric) *DataPoint {
+	metricLabels := makeLabels(m)
+	metricName := mf.GetName()
+	var metricValue interface{}
+
+	switch metricName {
+	case "kube_pod_status_phase":
+		metricValue = metricLabels["phase"]
+		delete(metricLabels, "phase")
+
+	// Only contribute labels
+	case "kube_pod_labels", "kube_pod_info", "kube_service_info", "kube_pod_container_info", "kube_node_labels":
+		metricValue = nil
+
+	// Formatted as Condition Values
+	case "kube_pod_status_ready", "kube_pod_status_scheduled", "kube_node_status_disk_pressure", "kube_node_status_memory_pressure", "kube_node_status_out_of_disk", "kube_node_status_ready":
+		if m.GetGauge().GetValue() == 1 {
+			metricValue = metricLabels["condition"]
```

Comment:
Question for you: it looks like this produces the string value `"kube_pod_status_ready": "true"`, for example. Would it be clearer to turn that into the boolean value `"kube_pod_status_ready": true`? Or is that just more confusing if we have `true`/`false`/`nil` (with `nil` for "unknown"?)